arch/arm/core/aarch32/mpu/Kconfig

```diff
@@ -82,6 +82,13 @@ config MPU_ALLOW_FLASH_WRITE
 	help
 	  Enable this to allow MPU RWX access to flash memory
 
+config MPU_DISABLE_BACKGROUND_MAP
+	bool "Disables the default background address map"
+	default n
```

Comment:
please remove `default n` as it is n by default

Reply:
Done